qiita_db/support_files/patches/47.sql

```diff
@@ -0,0 +1,404 @@
+        VALUES (bdiv_id, 'biom_table', 'artifact', True, NULL)
+        RETURNING command_parameter_id INTO bdiv_cp_id;
+    INSERT INTO qiita.command_parameter (command_id, parameter_name, parameter_type, required, default_value)
+        VALUES (arare_id, 'biom_table', 'artifact', True, NULL)
+        RETURNING command_parameter_id INTO arare_cp_id;
+    INSERT INTO qiita.command_parameter (command_id, parameter_name, parameter_type, required, default_value)
+        VALUES (srare_id, 'biom_table', 'artifact', True, NULL)
+        RETURNING command_parameter_id INTO srare_cp_id;
+
+    -- Step 4: Connect the artifact parameters with the artifact types that
+    -- they accept
+    SELECT artifact_type_id INTO biom_at_id
+        FROM qiita.artifact_type
+        WHERE artifact_type = 'BIOM';
+    INSERT INTO qiita.parameter_artifact_type (command_parameter_id, artifact_type_id)
+        VALUES (st_cp_id, biom_at_id),
+               (bdiv_cp_id, biom_at_id),
+               (arare_cp_id, biom_at_id),
+               (srare_cp_id, biom_at_id);
+
+    -- Step 5: Add the outputs of the command.
+    INSERT INTO qiita.command_output (name, command_id, artifact_type_id)
+        VALUES ('taxa_summary', sum_taxa_id, ts_at_id)
+        RETURNING command_output_id INTO ts_co_id;
+    INSERT INTO qiita.command_output (name, command_id, artifact_type_id)
+        VALUES ('distance_matrix', bdiv_id, dm_at_id)
+        RETURNING command_output_id INTO dm_co_id;
+    INSERT INTO qiita.command_output (name, command_id, artifact_type_id)
+        VALUES ('rarefaction_curves', arare_id, rc_at_id)
+        RETURNING command_output_id INTO rc_co_id;
+    INSERT INTO qiita.command_output (name, command_id, artifact_type_id)
+        VALUES ('rarefied_table', srare_id, biom_at_id)
+        RETURNING command_output_id INTO sr_co_id;
+
+    -- At this point we are ready to start transferring the data from the old
+    -- structures to the new structures. Overview of the procedure:
+    -- Step 1: Add initial set of artifacts up to rarefied table
+    -- Step 2: Transfer the "analisys jobs" to processing jobs and create
+    --         the analysis artifacts
+    -- Fun fact: after exploring the data on the database, we realized that
+    -- there are a lot of inconsistencies in the data. Unfortunately, this
+    -- makes the process of trasnferring the data from the old structure
+    -- to the new one a bit more challenging, as we will need to handle
+    -- different special cases.
+
+    -- Special case 1: there are jobs in the database that do not contain
+    -- any information about the options used to process those parameters.
+    -- However, these jobs do not have any results and all are marked either
+    -- as queued or error, although no error log has been saved. Since these
+    -- jobs are mainly useleess, we are going to remove them from the system
+    DELETE FROM qiita.analysis_job
```

Comment:
I _think_ these are the base for new analysis creation and done at User confirmation: https://github.com/biocore/qiita/blob/master/qiita_db/user.py#L320